source/Core/Drivers/Font.h

```diff
@@ -175,11 +175,17 @@ const uint8_t buttonB[] = {
 //     0xFE, 0x01, 0x79, 0x25, 0x79, 0x01, 0xFE, 0x00, 0x20, 0x20, 0x20, 0x20, 0xDF, 0x07, 0x8F, 0xDF, 0xFF, 0x01, 0xFE, 0x86, 0xDA, 0x86, 0xFE, 0x01,
 //     0x7F, 0x80, 0xA4, 0xBE, 0xA0, 0x80, 0x7F, 0x00, 0x04, 0x0E, 0x1F, 0x04, 0xFB, 0xFB, 0xFB, 0xFB, 0xFF, 0x80, 0x7F, 0x5B, 0x41, 0x5F, 0x7F, 0x80};
 
-const uint8_t infinityIcon[] = {
-    // width = 24
+const uint8_t infinityOnce[] = {

```

Comment:
Just a tiny naming suggestion. Since the base for the icons is not infinity sign anymore but repeat circle, I would suggest to name it for a better reflection of the change, like `RepeatOnce` for this variable...